testing/security/cipher_suites.py

```diff
@@ -262,9 +262,15 @@
 RFC_TO_OPENSSL_NAMES = sdk_utils.invert_dict(OPENSSL_TO_RFC_NAMES)
 
 
+def missing_ciphers(openssl_ciphers: set) -> bool:
```

Comment:
The return value is also a set and not a bool.

Reply:
Oops, yes.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2518/commits/e3e85bfd0521aaa4b9edcd78a6b268db20384981